webdav/server.go

```diff
@@ -43,14 +43,14 @@ func main() {
 	)
 
 	flag.IntVar(&port, "port", 8079, "this server's port")
-	flag.StringVar(&proxy, "proxyurl", "127.0.0.1:8080", "dfc proxy's url (ip:port)")
-	flag.StringVar(&tmpDir, "tmpdir", "/tmp/dfc", "temporary directory to store files used by webdav")
-	flag.IntVar(&logLevel, "loglevel", logLevelNone, "verbose level(0 = none, 1 = webdav, 2 = dfc)")
+	flag.StringVar(&proxy, "proxyURL", "127.0.0.1:8080", "dfc proxy's URL (ip:port)")
+	flag.StringVar(&tmpDir, "tmpDir", "/tmp/dfc", "temporary directory to store files used by webdav")
+	flag.IntVar(&logLevel, "logLevel", logLevelNone, "verbose level(0 = none, 1 = webdav, 2 = dfc)")
 	flag.Parse()
-	url := url.URL{Scheme: "http", Host: proxy}
+	dfcURL := url.URL{Scheme: "http", Host: proxy}
```

Comment:
url is very much locally used and golfing encourage's short local names, making it dfcURL doesn't add much value.

Reply:
The variable name `url` here conflicts with the import package with the same name.

Reply:
really? that is strange, it is a local variable, conflict with which one?